native/packages/react-native-bpk-component-banner-alert/src/style/index.android.js

```diff
@@ -0,0 +1,65 @@
+import {
+  colorGray50,
+  spacingBase,
+  spacingMd,
+  spacingSm,
+  spacingLg,
+  lineHeightXs,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import commonStyle from './style.common';
+
+const androidStyles = Object.assign({},
+  commonStyle, {
+    outerContainerSuccess: {
+      borderColor: colorGray50,
+    },
+    outerContainerWarn: {
+      borderColor: colorGray50,
+    },
+    outerContainerError: {
+      borderColor: colorGray50,
+    },
+    outerContainerNeutral: {
+      borderColor: colorGray50,
+    },
+    outerContainer: Object.assign({},
+      commonStyle.outerContainer,
+      {
+        backgroundColor: colorGray50,
+        borderWidth: 0,
+      }),
+    bannerContainerPadded: Object.assign({},
+      commonStyle.bannerContainerPadded,
+      {
+        backgroundColor: colorGray50,
+        paddingBottom: 0,
+        paddingLeft: 0,
+        paddingRight: 0,
+      }),
+    bannerContainerPaddedDismissable: Object.assign({},
+      commonStyle.bannerContainerPaddedDismissable,
+      {
+        paddingRight: spacingMd,
+      }),
+    bannerContainer: Object.assign({},
+      commonStyle.bannerContainer,
+      {
+        backgroundColor: colorGray50,
+        minHeight: 0,
+      }),
+    text: Object.assign({},
+      commonStyle.text,
+      {
+        paddingTop: spacingBase,
+        paddingBottom: spacingBase,
+        paddingRight: spacingLg,
+        lineHeight: lineHeightXs,
```

Comment:
@jamesf3rguson when we spoke offline the total height should be 48, however having `spacingBase` top and bottom (16 * 2) and the lineHeight for the base text using the default the total value was 64, I've used `lineHeightXs` that brings the total value to `50` will that do?